### DISCUSSION TITLE
Which release for Raspberry

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#689] [General]
Hi, i wish to know which release to be used for rpi3b*: *******.armhf.rpm or ******.armhf.deb , *****.linux.armhf.tar.gz ?
thanks

**Comment by oleg68** [comment]
Raspbian is a debian-based system, so you need  ******_armhf.deb  and  ******_all.deb for installation.

Having installed one GO version, you can simple unpack a a new one from ****.linux.armhf.tar.gz and test it without installation. This is mostly used temporary. If the new version looks well then it is better to download and install .deb files.

**Reply by paolo2504** [reply]
Raspbian is a debian-based system, so you need ******_armhf.deb and ******_all.deb for installation.
sudo apt install ./grandorgue_0.3.1.2341-14.os_armhf.deb won't work. error on dependencies as below. I've also downloaded ***_all, ***.resources and **.demo in same dir.

error says grandorgue-resources is not installed..


Having installed one GO version, you can simple unpack a a new one from ****.linux.armhf.tar.gz and test it without installation. This is mostly used temporary. If the new version looks well then it is better to download and install .deb files.
Pardon, i don't understand this point.... It seems ***.tar.gz is a compressed version of GO "as it were already installed"... Does this mean i can unpack it and launch GO as well ? (only after having installed it first). This is what i understand...

**Comment by paolo2504** [comment]
Definitely, which is the correct way to install go on raspberry os (probably the same under any other linux..).

download *.armhf, *.demo, *.os_all.
sudo apt install *.armhf.deb

However this way i get dependencies error. 
Firstly, it says 'grandorgue' is used instead of grandorgue_0.3.1.2341-14.os_armhf.deb
Then it says it needs grandorgue-resource that is not installable
Then it says it needs grandorgue-demo that is not installable
Finally it says wx3.0-i18 is needed but cannot be installed

Tried also with dpkg with similar errors.

**Comment by paolo2504** [comment]
and using gdebi i get this log:

W: grandorgue: package-name-doesnt-match-sonames libGrandOrgueCore0.3.1.2341 libGrandOrgueImages0.3.1.2341
E: grandorgue: debian-changelog-file-missing
E: grandorgue: no-copyright-file
E: grandorgue: description-starts-with-package-name
E: grandorgue: extended-description-is-empty
E: grandorgue: maintainer-name-missing osamarin68@gmail.com
E: grandorgue: manpage-not-compressed usr/share/man/man1/GrandOrgue.1
W: grandorgue: binary-without-manpage usr/bin/GrandOrgueTool
E: grandorgue: shlib-with-executable-bit usr/lib/libGrandOrgueCore.so.0.3.1.2341 0755
E: grandorgue: shlib-with-executable-bit usr/lib/libGrandOrgueImages.so.0.3.1.2341 0755
W: grandorgue: maintscript-calls-ldconfig postinst
E: grandorgue: package-must-activate-ldconfig-trigger usr/lib/libGrandOrgueImages.so.0.3.1.2341
W: grandorgue: maintscript-calls-ldconfig postrm

Lintian terminato con codice di uscita 1

**Comment by paolo2504** [comment]
Sorry give up: i installed first file resources and then GO can be installed.

**Comment by oleg68** [comment]
Please take a look at https://github.com/oleg68/GrandOrgue/blob/master/INSTALL.md for the step-by-step guide.

**Comment by paolo2504** [comment]
Thanks Oleg : I fixed on my own by installing first the resources then GO main. 
Regards
